packages/tools/src/Tool.ts

```diff
@@ -97,7 +97,7 @@ export interface MonorepoRoot {
   /**
    * The underlying tool implementation for this monorepo.
    */
-  tool: Tool;
+  tool: string;
```

Comment:
this is a breaking change, right?

Reply:
Yeah, I listed it as a major in the changeset.